dockerfiles/dataset/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+MAINTAINER Madhuri Yechuri <madhuri.yechuri@clusterhq.com>
+
+# Note: apt-get install requires the `-1` (epoch) suffix.
+ENV FLOCKER_VERSION 1.14.0-1
+
+RUN sudo apt-get update
+RUN sudo apt-get -y install apt-transport-https software-properties-common
+RUN sudo add-apt-repository -y ppa:james-page/docker
```

Comment:
Don't need docker or the docker repo for this image. 
(definitely not that ppa repo)